.travis.yml

```diff
@@ -16,3 +16,10 @@ notifications:
   email:
     on_success: never
     on_failure: always
+
+after_success:
```

Comment:
In the script section, we can call `npm run coverage` (and setup this command correctly) to avoid after_success script